.github/workflows/test.yml

```diff
@@ -2,18 +2,19 @@ name: Test
 on: [push]
 env: 
   CARGO_TERM_COLOR: always
+  RUST_VERSION: 1.56.1
```

Comment:
While this PR was open, it seems master upgraded to 1.60 and this PR brings it back down (probably not on purpose).

Reply:
maybe updating from main is the thing to do.